lesson_15/edmond_sabou/test_home1_les13.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+
+
+class TestPoligons(unittest.TestCase):
+
+    def test_if_side_is_not_none(self):
+        sides = 1, "asd", 3
```

Comment:
the test is good, the only problem is that the code is not used in this test. You need to make an instance of Polygon with the sides you defined and make the asserts on the instance_of_poligon.sides

Reply:
I had a feeling something was not ok. changed it now. please check